polls/models.py

```diff
@@ -17,8 +16,16 @@ def choices(self):
 
 
 class Choice(Page):
-    votes = models.IntegerField(default=0)
```

Comment:
I changed it because the names could be very confusing, but will put a comment in and leave it as is